src/rest-client.ts

```diff
@@ -103,8 +103,14 @@ import {
   AmendAlgoOrderRequest,
   AmendAlgoOrderResult,
   AlgoOrderRequest,
+  EconomicCalendarData,
+  UnitConvertData,
 } from './types';
 import { ASSET_BILL_TYPE } from './constants';
+import {
+  EconomicCalendarRequest,
+  UnitConvertRequest,
+} from './types/rest/request/public';
```

Comment:
Take a look at the index.ts in `./types/rest/request`, if you update it to also export public.ts, you'll be able to import this neatly from `./types`

Reply:
Ready to merge from my side

Reply:
Now that public.ts is exported, you can move both `EconomicCalendarRequest` and `UnitConvertRequest` to the import from './types'